### PR TITLE
Itea documentation

### PR DESCRIPTION
**Pull request by UkrainianEngineer**
Updated documentation format.
You may see how it looks generated in my `IT-documentation` branch.

### CODE REVIEW

File: docs/procedures/working_with_branches.rst [hunk 1]
```diff
@@ -37,19 +41,27 @@ Aggregated file should be commited and merged.
...
+
 After tagging branch it may be removed.
 
 Remove remote branch:
+
     `git push origin --delete <branch_name>`
 
 Remove local branch:
+
     `git branch -d <branch_name>`
```
Comment (Sasha1152) [inline docs/procedures/working_with_branches.rst:76]: Add, please, instruction for create new branch from release

Comment (UkrainianEngineer) [reply docs/procedures/working_with_branches.rst:76]: Added.

**Comment by UkrainianEngineer** [comment]
@Sasha1152 , @kate-ka , @RomanPryima  guys please review this pull request.

**Review by kate-ka — APPROVED** [review]
👍

**Pull request merged**